src/plugins/intel_gna/src/log/gna_plugin_log.hpp

```diff
@@ -16,8 +16,6 @@
 #define PLOT
```

Comment:
I understand that log level is now fully controlled from command line.
De you think we need all this defines?

Reply:
no it should be refactored but separately